core/tests/protractor_desktop/playVoiceovers.js

```diff
@@ -0,0 +1,104 @@
+
+var ExplorationEditorPage = require(
+  '../protractor_utils/ExplorationEditorPage.js');
+var CreatorDashboardPage = require(
+  '../protractor_utils/CreatorDashboardPage.js');
+var ExplorationPlayerPage = require(
+  '../protractor_utils/ExplorationPlayerPage.js');
+var LibraryPage = require('../protractor_utils/LibraryPage.js');
+
+describe('Voiceover player', function() {
+  var creatorDashboardPage = null;
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorTranslationTab = null;
+  var explorationEditorSettingsTab = null;
+  var explorationPlayerPage = null;
+  var libraryPage = null;
+
+  beforeAll(async function() {
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationEditorTranslationTab =
+      explorationEditorPage.getTranslationTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    libraryPage = new LibraryPage.LibraryPage();
+
+    await users.createUser('testVoiceovers@voiceovers.com',
+      'testVoiceovers');
+    await users.login('testVoiceovers@voiceovers.com');
+    await workflow.createExploration();
+    await explorationEditorMainTab.exitTutorial();
+    await explorationEditorMainTab.setStateName('First');
+    await explorationEditorMainTab.setContent(await forms.toRichText(
+      'This is the first card.'));
+    await explorationEditorMainTab.setInteraction('EndExploration');
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Hindi', '../data/cafe.mp3');
+    await explorationEditorPage.saveChanges();
+    await explorationEditorPage.navigateToSettingsTab();
+    await explorationEditorSettingsTab.setTitle('voiceoverPlayerTest');
+    await explorationEditorSettingsTab.setCategory('Languages');
+    await explorationEditorSettingsTab.setLanguage('English');
+    await explorationEditorSettingsTab.setObjective(
+      'Testing if voiceovers work');
+    await explorationEditorPage.saveChanges('Done.');
+    await workflow.publishExploration();
+    await users.logout();
+  });
+
+  it('should play and pause voiceovers', async function() {
+    await users.login('testVoiceovers@voiceovers.com');
+    await libraryPage.get();
+    await libraryPage.playExploration('voiceoverPlayerTest');
+    await explorationPlayerPage.expandAudioBar();
+    await explorationPlayerPage.pressPlayButton();
+    await explorationPlayerPage.expectAudioToBePlaying();
+    await explorationPlayerPage.pressPauseButton();
+    await explorationPlayerPage.expectAudioToBePaused();
+  });
+
+  it('should play voiceovers for multiple languages', async function() {
+    await creatorDashboardPage.get();
```

Comment:
this it blocks depends on the upper it block, right?

@U8NWXD, Do we allow having dependent `it` blocks?

Reply:
We don't have a strict prohibition on dependent `it` blocks, but we'd like to avoid them whenever possible. @krishnarao22 is there a reason for these blocks to be dependent?

Reply:
I've tried to make the `it` blocks more independent. I've made the user login and log out before and after each `it` block. 

I've also completely moved the creation of the exploration to the `beforeAll` block (so that we're not adding translations to the exploration within the `if` block.

Please let me know if I need to make the blocks more independent!